libcrux-ml-dsa/src/simd/portable/arithmetic.rs

```diff
@@ -350,10 +369,12 @@ pub(super) fn power2round(t0: &mut Coefficients, t1: &mut Coefficients) {
 // TODO: Revisit this function when doing the range analysis and testing
```

Comment:
Is this still an open todo?

Reply:
Still todo until we analyze all call-sites